### ISSUE TITLE
`alchemy`: improve performance of `status` with process pool executor

### ISSUE THREAD

**Issue by hmacdope** [asapdiscovery/asapdiscovery#899]
We can improve the perfomance of `status` CLI command by having queries to client run in parallel using a `ProcessPoolExecutor`. @dotsdl can provide further guidance 

```python
        for key in running_networks: # -> parallelism over process
            # get status
            network_status = client._client.get_network_status(
                network=key, visualize=False
            )
            running_tasks = client._client.get_network_actioned_tasks(network=key)
            if (
                "running" in network_status or "waiting" in network_status
            ) and running_tasks:
                table.add_row(
                    str(key),
                    str(network_status.get("complete", 0)),
                    str(network_status.get("running", 0)),
                    str(network_status.get("waiting", 0)),
                    str(network_status.get("error", 0)),
                    str(network_status.get("invalid", 0)),
                    str(network_status.get("deleted", 0)),
                    str(len(running_tasks)),
                )
        status_breakdown.stop()
        console.print(table)

```

This also applies to proposed slackbot functionality in https://github.com/asapdiscovery/FALCBot/issues/6

**Comment by hmacdope** [comment]
May be better to package this functionality up in its own function so that we don't have to re-implement the same API surface in `FALCBot` as well.

**Comment by dotsdl** [comment]
@hmacdope I've got an upstream solution articulated here: https://github.com/openforcefield/alchemiscale/issues/251

In the meantime, I'll take on addressing this in the short term with process parallelism in `asap-alchemy`.

**Comment by dotsdl** [comment]
Since we're already working on the upstream solution, with an aim to deploy in the next couple of weeks, I propose we avoid expending effort on this workaround. Is that okay @hmacdope?

**Comment by hmacdope** [comment]
Absolutley, happy to wait for upstream.

**Comment by dotsdl** [comment]
Returning to this now; partially addresses #985.

**Comment by dotsdl** [comment]
This will be closed by #1011.
